RoslynSecurityGuard/RoslynSecurityGuard/Messages.yml

```diff
@@ -53,6 +53,12 @@ SG0009:
   title: The cookie is missing security flag HttpOnly
   description: It is recommended to specify the HttpOnly flag to new cookie.
 
+# XSS
+SG1111:
```

Comment:
I will change it to SG0027. This is find to push PR with temporary ID to avoid collision.